spec.bs

```diff
@@ -1015,7 +1088,7 @@ To <dfn>score and rank a bid</dfn> given an [=auction config=] |auctionConfig|,
   1. Set |bidValue| to |generatedBid|'s [=generated bid/modified bid=].
 1. Let |owner| be |generatedBid|'s [=generated bid/interest group=]'s [=interest group/owner=].
 1. Let |browserSignals| be an [=ordered map=] whose [=map/keys=] are [=strings=] and whose
-  [=map/values=] are {{any}}.
+  [=map/values=] are {{any}}. TODO: Change to an IDL record<>.

```

Comment:
Can you make this a {{BiddingBrowserSignals}} now that you've defined that, or are there other fixups needed that should wait for another PR?

Reply:
fixed in [PR 652](https://github.com/WICG/turtledove/commit/b67ce358d7fb1347e6c0800a6a77aa6599c7a991)